File: src/Projects/Android/016-TextGeneratorSystemNetLib/Server/src/main/java/org/csystem/app/generator/text/server/configuration/thread/ThreadPoolConfig.java
package org.csystem.app.generator.text.server.configuration.thread;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;

@Configuration
public class ThreadPoolConfig {
    @Bean
    public ExecutorService createThreadPool()
    {
        return Executors.newCachedThreadPool();
    }
}

File: src/Projects/Android/ChatSystem/Server/src/main/java/org/csystem/app/chat/server/configuration/thread/ThreadPoolConfig.kt
package org.csystem.app.chat.server.configuration.thread

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.util.concurrent.ExecutorService
import java.util.concurrent.Executors

@Configuration
open class ThreadPoolConfig {
    @Bean
    open fun createThreadPool(): ExecutorService
    {
        return Executors.newCachedThreadPool()
    }
}
